<?xml version="1.0"?>
<launch>
    <!-- Тип автопилота -->
    <!-- 0 - PX4 -->
    <!-- 1 - ArduPilot -->
    <arg name="ap_type" default="0" />
    <!-- Высота взлета над землей в метрах -->
    <arg name="altitude" default="1.5" />
    <!-- Использовать визуальную одометрию или нет -->
    <arg name="use_vision_odometry" default="0" />
    <!-- Количество кругов поворота: положительное число -->
    <arg name="laps" default="3" />
    <!-- Угловая скорость в рад/с, может быть больше и меньше нуля -->
    <arg name="ang_vel" default="0.7"/>

    <node name="$(anon spin_laps_test)" pkg="flight_tests" type="spin_laps_test.py" output="screen">
        <param name="altitude" value="$(arg altitude)" />
        <param name="autopilot_type" value="$(arg ap_type)" type="bool"/>
        <param name="use_vision_odometry" value="$(arg use_vision_odometry)" type="bool"/>
        <param name="laps" value="$(arg laps)"/>
        <param name="ang_vel" value="$(arg ang_vel)"/>
    </node>
</launch>
